### ISSUE TITLE
Plane Sensor not working on Android Devices

### ISSUE THREAD

**Issue by Sharian** [x3dom/x3dom#423]
Hi, 

the following example is not working on android devices: 
doc.x3dom.org/tutorials/animationInteraction/transformations/example.html

Cheers

**Comment by Wolsten** [comment]
This is as also an issue on iPhones which seems like a big issue in terms of interactivity. Any news on whether this is likely to be provided or are there alternative approaches for use on mobiles? Thanks

**Comment by mlimper** [comment]
There are two ways to get this working:

* Implement touch support for sensor nodes

* Implement your own touch event listeners on the application side. Here is an example that uses DOM mouse events to implement a dragging behavior on the application side, this should work in a similar fashion for touch: http://www.x3dom.org/x3dom/example/MovingObjectsWithDOMEvents.html

**Comment by Wolsten** [comment]
Thanks for the feedback Max. I tried that example, but it locks up on my iPhone6:-( 

I am not sure if I am up to implementing touch support myself - is it not there yet because not many people are interested in it or is it just intrinsically difficult to implement? Its lack is a showstopper for me at present which is a shame as otherwise I have really enjoyed getting to grips with it as an amateur in 3D terms.

**Comment by Wolsten** [comment]
Ah, sorry, having re-read what you wrote, you meant that the example would also have to be recoded using touch events?

**Comment by mlimper** [comment]
> Ah, sorry, having re-read what you wrote, you meant that the example would also have to be recoded using touch events?

yes, for the application you need - the point of the example is that it (hopefully) shows that it is actually not too complicated to implement something like this dragging functionality for your application (although it involves a bit of math, but that's already included in the example code and ready to be used)

> I am not sure if I am up to implementing touch support myself - is it not there yet because not many people are interested in it or is it just intrinsically difficult to implement?

Neither the one reason nor the other one, I think the main reason is that simply no one took the time yet to sit down and do it ;-)
If someone is willing to implement this for the pointing device sensor nodes, following the code that is already there for mouse events, this would of course be a great contribution to X3DOM.

**Comment by Wolsten** [comment]
I am getting the example working with not too much difficulty thanks.

**Comment by mlimper** [comment]
Cool, thanks for the update

**Comment by khanh2003** [comment]
`pinging again` - you should sit down and work on it.
=======================================

**Comment by mlimper** [comment]
Everyone is welcome to contribute!
